crates/re_ui/src/list_item2/mod.rs

```diff
@@ -0,0 +1,76 @@
+//! Second-generation list item.
+//!
+//! TODO(ab): provide some top-level documentation here.
+
+mod label_content;
+mod list_item;
+mod other_contents;
+mod scope;
+
+pub use label_content::*;
+pub use list_item::*;
+pub use other_contents::*;
+pub use scope::*;
+
+/// Context provided to [`ListItemContent`] implementations
+#[derive(Debug, Clone)]
+pub struct ContentContext<'a> {
+    /// Area available for the content to draw in.
+    pub rect: egui::Rect,
+
+    /// Background area
+    pub bg_rect: egui::Rect,
```

Comment:
How does this relate to `Self::rect`?

Reply:
This is the rectangle for the full span highlight. Useful eg to test for the cursor. I'll clarify the docstring.